test_schema.py

```diff
@@ -0,0 +1,54 @@
+# Rikki 6/4/20
```

Comment:
One benefit of git is you also get a history of who added/changed which lines and when. It bypasses the need for comments like these. I can show you how to see that when we next sync up.